src/webpages/details.js

```diff
@@ -35,14 +35,13 @@ function Details() {
 
   if (detail) {
     return (
-      <div className="container">
-        <div className="row d-flex gap-3 p-5">
-          <div className="col-sm-8">
-            <img src={detail.image_path} className="thumbnail img-responsive img-fluid rounded mx-auto d-block h-75 w-75" alt="house comes here" />
+      <div className="container-fluid">
+        <div className="row d-flex p-5">
+          <div className="col-sm-8 h-50">
+            <img src={detail.image_path} className="thumbnail img-responsive img-fluid rounded mx-auto d-block h-50 w-50" alt="house comes here" />
           </div>
-          {/* <div className="col-sm-2" /> */}
-          <div className="col-sm-3">
-            <p className="fs-3">{detail.address}</p>
+          <div className="col-sm-3 h-50">
+            <p className="fs-3 text-end">{detail.address}</p>
             <table className="table table-striped">
               <tbody>
                 <tr>
```

Comment:
- [ ] Your container doesn't cover the page vertically. Kindly fix it.
![image](https://user-images.githubusercontent.com/91486791/204767600-cec1dbaa-3b59-4b62-859b-62bcc9a27d55.png)
![image](https://user-images.githubusercontent.com/91486791/204767944-5ed7f9ad-0e26-46fc-bf2f-d4f6bec60d28.png)

Reply:
@tugsem Ok I will make adjustments.